packages/aquarius/src/bot/plugins/rss.js

```diff
@@ -75,13 +75,19 @@ async function checkForUpdates(guild, url, name, analytics) {
     log(error);
     Sentry.captureException(error);
 
+    const errorChannel = guild.channels.cache.find((c) => c.name === errorName);
+
+    if (!errorChannel) {
+      errorChannel = channel;
+    }
```

Comment:
Since it's declared a `const` this will actually break - not sure why the linter isn't running on external pull requests, but will look into that!

```javascript
const errorChannel = guild.channels.cache.find((chan) => chan.name === errorName) ?? channel
```

Should work, or you can change `const` to `let`